### PR TITLE
Refactor X trial pipeline and add pipeline duration to output

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- Directly call the x provision pipeline from the main Pipeline Runner
- Adding duration to the Pipeline runner output
- Format pipeline runner output in GitHub issues/comments as table for better readability

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/apigee-x-trial-provision/apigee-x-trial-provision.sh [hunk 1]
```diff
@@ -56,14 +56,14 @@ esac
...
 
-
-if ! [ -x "$(command -v jq)" ]; then
-  >&2 echo "ABORTED: Required command is not on your PATH: jq."
-  >&2 echo "         Please install it before you continue."
-
-  exit 2
-fi
-
+for dependency in jq openssl
```
Comment (laughingbiscuit) [inline tools/apigee-x-trial-provision/apigee-x-trial-provision.sh:59]: i have always used `which` for this in the past - whats the difference?

File: tools/apigee-x-trial-provision/pipeline.sh [hunk 2]
```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-gcloud builds submit --config ./cloudbuild.yaml \
-  --substitutions="_DELETE_AFTER_VALIDATION=true,_CERTIFICATES=generated" \
-  --project "$X_TRIAL_GCP_PROJECT"
+set -e
+
+./apigee-x-trial-provision.sh --quiet --certificates generated --project "$X_TRIAL_GCP_PROJECT"
```
Comment (laughingbiscuit) [inline tools/apigee-x-trial-provision/pipeline.sh:19]: please run relative to `SCRIPTPATH` as we do in other files

File: tools/pipeline-runner/run-pipelines.sh [hunk 3]
```diff
@@ -25,32 +25,38 @@ run_single_pipeline() {
...
+    run_single_pipeline "$DIR"
+    PIPELINE_EXIT=$?
+    ENDTIME=$(date +%s)
+    PIPELINE_REPORT="$PIPELINE_REPORT;$DIR,$PIPELINE_EXIT,$((ENDTIME-STARTTIME))s"
+  fi
+done
+
 # print report
-echo "$PIPELINE_REPORT" | tr ";" "\n" | awk -F"," '$2 = ($2 > 0 ? "fail" : "pass")' OFS=";" | column -s ";" -t > pipeline-result.txt
+echo "$PIPELINE_REPORT" | tr ";" "\n" | awk 'NF' | awk -F"," '$2 = ($2 > 0 ? "fail" : "pass")' OFS=";" > pipeline-result.txt
```
Comment (laughingbiscuit) [inline tools/pipeline-runner/run-pipelines.sh:56]: why dont we like `column` anymore? if you are making additional changes not related to the pr (`X trial remove cloud build`) please can you put a description of what you were doing so we can review properly? I wasn't expecting any changes to the `pipeline-runner` dir in this PR

**Review by laughingbiscuit — APPROVED** [review]

File: tools/apigee-x-trial-provision/apigee-x-trial-provision.sh (hunk 1 above)
Comment (danistrebel) [reply tools/apigee-x-trial-provision/apigee-x-trial-provision.sh:59]: https://unix.stackexchange.com/a/85250 is a better explanation than we probably needed 🤓 In our case the two will behave the same as both are built in commands

File: tools/pipeline-runner/run-pipelines.sh (hunk 3 above)
Comment (danistrebel) [reply tools/pipeline-runner/run-pipelines.sh:56]: I moved the column down to the print. The reformatting of the output is related to this PR as I felt the need to add time time tracking to such long running pipelines. Adding this to the PR description now.

**Review by laughingbiscuit — APPROVED** [review]

**Pull request merged**